lib/bundle/bundle.rb

```diff
@@ -33,8 +33,7 @@ def mas_signedin?
 
   def cask_installed?
     @cask_installed ||= begin
-      File.directory?("#{HOMEBREW_PREFIX}/Caskroom") &&
-        File.directory?("#{HOMEBREW_REPOSITORY}/Library/Taps/caskroom")
```

Comment:
This shouldn't be changed; we don't want running `brew cask list` to automatically tap this.